pkg/controllers/dynakube/extension/databases/deployment.go

```diff
@@ -0,0 +1,254 @@
+package databases
+
+import (
+	"context"
+	"fmt"
+	"maps"
+	"slices"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/latest/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/latest/dynakube/extensions"
+	"github.com/Dynatrace/dynatrace-operator/pkg/consts"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/labels"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	probePort = int32(8080)
+	// Keep in sync with helm chart
+	defaultServiceAccount = "dynatrace-database-extensions-executor"
+	// Must contain the ID specified in the DynaKube CR.
+	executorLabelKey = "extensions.dynatrace.com/executor.id"
```

Comment:
```suggestion
	executorIDLabelKey = "extensions.dynatrace.com/executor.id"
```
Not sure if it shold be "Id" or "ID", though.